testui/support/appium_driver.py

```diff
@@ -411,26 +413,21 @@ def __local_run(url, desired_caps, use_port, udid, log_file):
         if udid is None:
             desired_caps = __set_android_device(desired_caps, device)
         logger.log(f'setting device for automation: {desired_caps["udid"]}')
-        root_dir = (
-            os.path.dirname(
-                os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-            )
-            + "/"
-        )
-        Path(root_dir + "appium_logs").mkdir(parents=True, exist_ok=True)
+        root_dir = os.path.join("./logs", "appium_logs")
+        Path(root_dir).mkdir(parents=True, exist_ok=True)
         file_path: str
         if log_file == "appium-stdout.log":
-            file = f"appium_logs/testui-{udid}-" + log_file
+            file = os.path.join(root_dir, f"testui-{udid}-{time.time()}-" + log_file)
         else:
-            file = f"appium_logs/{log_file}"
-        with open(root_dir + file, "wb") as out:
+            file = os.path.join(root_dir, log_file)
+        with open(file, "wb") as out:
             process = subprocess.Popen(
                 ["appium", "-p", port.__str__(), "-bp", bport.__str__()],
                 stdout=out,
                 stderr=subprocess.STDOUT,
             )
             atexit.register(process.kill)
-        file_path = root_dir + file
+        file_path = file
```

Comment:
At this point would be cleaner and more accurate to just replace any uses of the `file` variable with `file_path`

Reply:
Yes Ill do